### ISSUE TITLE
Delete a Wishlist

### ISSUE THREAD

**Issue by asg8830** [CSCI-GA-2820-FA24-001/wishlists#4]
**As a** customer
**I need** the ability to delete an existing wishlist
**So that** I can remove a wishlist that is no longer needed
   
 ### Details and Assumptions
 * The list that is being deleted exists in the database
 * There is a message confirming that the delete was completed
 * Should land on the view wishlist page once delete is done
   
 ### Acceptance Criteria  
   
 ```gherkin
 Given a customer has at least one wishlist
 When the user wants to delete a wishlist
 Then the user should be remove the wishlist from the database
```
